lib/workload/stateless/stacks/stacky-mcstackface/glue-constructs/jb-weld/part_3/fastq-list-row-event-shower-complete-to-cttsov2-ready/step_functions_templates/fastq_list_row_shower_complete_event_to_cttsov2_draft_events_sfn_template.asl.json

```diff
@@ -11,25 +11,51 @@
     },
     "Get Libraries from Instrument Run": {
       "Type": "Task",
-      "Resource": "arn:aws:states:::dynamodb:getItem",
+      "Resource": "arn:aws:states:::aws-sdk:dynamodb:scan",
```

Comment:
Just a note.

This is ok. Perhaps, create GSI/LSI indexes on those none PK/SK lookup attributes where appropriate.
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SecondaryIndexes.html

If, however, the use case started to explode such that the need of `select * from table` use case all the time or, some dynamic with filtering and predicates condition to a table then it is ok to tap into Postgres.

Anyhow. We shall observe this part as-is for now.

Reply:
This is runs one per instrument run, I don't think this is a burden on our compute resources